Artconomy/short_stuff
# WIP: Sane default for primary key, fix default value in admin

Describe what this pull request changes, and why. If there is a big design decision(s) involved, put it in the codebase (docstring/README/ADR) and link to it here.

**JIRA tickets**: If there is a related publicly viewable JIRA ticket, mention that here. Also mention the OSPR ticket number here once it gets created (after you open the PR), if applicable. Otherwise omit this. OpenCraft JIRA ticket numbers should just be put at the beginning of the PR title in square brackets.

**Discussions**: Link to any past public discussions about this PR or the design/architecture. Otherwise omit this.

**Dependencies**: None

**Screenshots**: Always include screenshots if there is any change to the UI (ideally, both "before" and "after" screenshots, if applicable). If there are no UI changes, omit this.

**Sandbox URL**: TBD - sandbox is being provisioned.

**Merge deadline**: "None" if there's no rush, "ASAP" if it's critical, or provide a specific date if there is one.

**Testing instructions**:

1. Step by step manual setup/testing/verification instructions for your reviewers go here. Make sure you (the author) have already manually tested it using the same steps.
2. Step 2

**Author notes and concerns**:

1. List any concerns, known bugs, or open questions.
2. Are there any database migrations? 🚩

**Reviewers**
